app/models/organisation.rb

```diff
@@ -3,6 +3,9 @@ class Organisation < ApplicationRecord
   has_many :owned_case_logs, class_name: "CaseLog", foreign_key: "owning_organisation_id"
   has_many :managed_case_logs, class_name: "CaseLog", foreign_key: "managing_organisation_id"
 
+  include DbEnums
+  enum providertype: DbEnums.providertype, _suffix: true
```

Comment:
Think the `display_attributes` method here will also need updating since it refers to `org_type` still?

Reply:
Yes! Missed it, thanks!